entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -1184,3 +1129,532 @@ dotnet new ef-templates
 The templates can then be customize and will automatically be used by `dotnet ef dbcontext scaffold` and `Scaffold-DbContext`.
 
 For more details, see [Custom Reverse Engineering Templates](xref:core/managing-schemas/scaffolding/templates).
+
+> [!TIP]
+> The EF Team demonstrated and talked in depth about reverse engineering templates in an episode of the .NET Data Community Standup. As with [all Community Standup episodes](https://aka.ms/efstandups), you can [watch the T4 templates episode now on YouTube](https://youtu.be/x2nh1vZBsHE).
+
+## Model building conventions
+
+EF Core uses a metadata "model" to describe how the application's entity types are mapped to the underlying database. This model is built using a set of around 60 "conventions". The model built by conventions can then be customized using [mapping attributes (aka "data annotations")](xref:core/modeling/index#use-data-annotations-to-configure-a-model) and/or calls to the [`DbModelBuilder` API in `OnModelCreating`](xref:core/modeling/index#use-fluent-api-to-configure-a-model).
```

Comment:
DbModelBuilder? can link to API reference too.

Reply:
This is a link to the conceptual docs, which seems more useful here than a link to the API docs.